src/entities/Profile.ts

```diff
@@ -20,8 +29,11 @@ export default class Profile {
 	@OneToOne(() => User, user => user.profile)
 	public user!: User;
 
-	@Column()
-	@IsString()
+	@Column({
+		'type': 'enum',
+		'enum': Course,
+		'default': Course.COMPUTER_SCIENCE
+	})
 	public course!: string;
```

Comment:
Can you add back the `@IsString()` decorator? It's used for validation before we trying saving the profile to the database (just so we don't try to unnecessarily save when the data is garbage)